### PR TITLE
[algorithmPaintTlbx] cursor only in view & clean of interpolate

### PR DESCRIPTION
**Pull request by mathildemerle**
:icecream: 

> 1° Paint cursor only displayed in the view !
> 2° Cursor more accurate
> 3° Small cleaning in interpolate
> 4° Adapted cursor to multiple tabs

:ice_cream:

### CODE REVIEW

**Comment by LoicCadour** [comment]
Concerning the cursor, I don't see the changes.
Correct me if I am wrong but the *mouseMoveEvent* is on the view so not used when hovering the rest of the app.
I can suggest you to look at what has been done in medVtkView.cpp: take the widget of the view, set its cursor, etc.
Could work.

**Comment by mathildemerle** [comment]
The size of the cursor is more accurate.

I do not understand what you are saying. It's working in mouseMoveEvent. Everything is fine on my computer. No problem for me. The cursor is displayed only in the viewer.

**Comment by LoicCadour** [comment]
Well I triple-checked that I was on the right branch, and it still doesnt work on my computer.
I put a qDebug() in the part where you Manage cursors (at line 151). And the debug message only appears when hovering the view, so the cursor is never disabled. That's at least what's happening on my computer.

**Comment by mathildemerle** [comment]
Used an other way to change the cursor. Adapted the cursor to multiple tabs also. 

Can you test that loic on Windows ? :)

**Comment by LoicCadour** [comment]
OK fine for me.
Another TESTER please (yeah TESTER was in capital, and written twice)

**Comment by mathildemerle** [comment]
It would be great to test on mac also (floooo :p) because the previous problem was caused as far as we know by differences between linux and windaub... windows (huhu).

Thx Loic \o

File: src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp [hunk 1]
```diff
@@ -54,6 +55,7 @@
 #include <set>
 
 #include <limits>
+#include <numeric>
```
Comment (LoicCadour) [inline src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp]: needed?

**Comment by fcollot** [comment]
I'm sorry there's more urgent problems Hubert needs me to solve right now.

**Comment by LoicCadour** [comment]
Works as well on Mac.I remove #include<numeric> and I merge it.

**Comment by mathildemerle** [comment]
Thx Loic :)

**Comment by LoicCadour** [comment]
It was already merged :(

**Pull request merged**